chips/sam4l/src/usart.rs

```diff
@@ -773,11 +818,13 @@ impl hil::spi::SpiMaster for USART {
 
     /// Pass in a None to use the HW chip select pin on the USART (RTS).
     fn specify_chip_select(&self, cs: Self::ChipSelect) {
+        self.enable_clock();
```

Comment:
We need to turn the clock in every time we touch the registers I think, though I hear your concern. Will look deeper